lib/server/proxyClientRequestToApi.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+var Backbone = require("../application/Backbone");
+var Configure = require("../application/Configure");
+
+function proxyClientRequestToApi(apiHost){
+    return function(req,response,next){
+        if (!!req.headers['brisket-client']){
+            Configure.backbone();
+            Backbone.ajax( apiHost + req.path)
+            .then(function(data) {
```

Comment:
I actually considered request, but decided on Backbone to make it as close as possible to the ServerApp implementation. I also liked that I could take advantage of the existing "promisification".

I can work on tests too. May need a little help.